### PR TITLE
improve CUPED page

### PR DESCRIPTION
**Pull request by schmit**

### CODE REVIEW

File: docs/statistics/cuped.md [hunk 1]
```diff
@@ -6,17 +6,53 @@ However, there are options beyond waiting to gather more data, and all these opt
...
 
-An illustration of how CUPED helps reduce the variance in an effect estimate, leading to a much tighter confidence interval.
+The above sketch illustrates how CUPED helps reduce the variance in an effect estimate, leading to a much tighter confidence interval.
 
-## Overview
+:::info
 
-When CUPED is enabled, Eppo automatically ingests aggregated data about each subjects' events across metrics that are added to your experiment for the 30 days before they joined the experiment. Eppo then uses a ridge regression model to predict subject-level outcomes for each metric. For each metric, we predict outcomes using every available metric. These predictions are then used to get more accurate effect estimates, reducing the variance and hence leading to better confidence intervals.
+Note that we only use CUPED to reduce variance on the metrics overview page.
+In particular, not that there are no CUPED estimates for:
```
Comment (emetelka) [inline docs/statistics/cuped.md:14]: > In particular, not that there are no CUPED estimates for:

This should be "note"

Can we add an explanation as to why we only focus on the metrics overview page?

File: docs/statistics/cuped.md [hunk 2]
```diff
@@ -6,17 +6,53 @@ However, there are options beyond waiting to gather more data, and all these opt
...
+
+## Model
+
+For each of the metrics in the experiment, we now regress the outcome on the covariates as defined above using a ridge regression, separately for each variation.
+We can then use the predictions of these models to improve the effect estimates, with the variance of these estimates being proportional to the mean square error of the predictions.
+Thus, the improvement in the confidence intervals is directly related to how well the pre-experiment data can predict experiment outcomes.
+
+CUPED works best for experiments with long-time users for whom many pre-experiment data points exist. It is generally less effective for newer users; if you are running an experiment on a change in the onboarding flow for new users, there is no prior data to leverage, just the assignment dimensions, and hence there is less benefit. However, one appealing characteristic of our approach that you do not have to worry about CUPED giving you worse results. In the worst case scenario, it does equally well as the standard approach.
+
+## What makes Eppo's CUPED++ different
```
Comment (emetelka) [inline docs/statistics/cuped.md:49]: This section didn't help me understand how CUPED++ is different. Are the drawbacks you point out related to a basic CUPED model that CUPED ++ does not have? Or are they about our model or CUPED generally?

Comment (schmit) [reply docs/statistics/cuped.md:49]: I give two points of difference below; what about it do you not understand?

File: docs/statistics/cuped.md [hunk 3]
```diff
@@ -51,8 +51,8 @@ CUPED works best for experiments with long-time users for whom many pre-experime
...
 
-- For some metrics, there is no clear pre-experiment equivalent: e.g. a conversion or retention metric. In our case, we can still leverage historical data of related metrics to help improve estimates
-- This approach does not help for experiments where no pre-experiment data exists (experiments on new users). Our approach is able to leverage assignment properties to reduce variance.
+- For some metrics, there is no clear pre-experiment equivalent for that metric: e.g. a conversion or retention metric. In our implementation, we can still leverage historical data of the other experiment metrics to help improve estimates of these conversion and retention metrics. This allows us to get improved estimates for conversion and retention metrics versus a standard CUPED approach.
+- CUPED help for experiments where no pre-experiment data exists (experiments on new users, such as onboarding flows). Because we also use assignment properties as covariates in the regression adjustments model, we are able to reduce variance for these experiments as well, which leads to smaller confidence intervals for such experiments.
```
Comment (emetelka) [inline docs/statistics/cuped.md:55]: Wording, change to:
CUPED helps experiments

**Pull request merged**